src/views/open-api-docs-view/components/property-details/property-details.module.css

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/*
+
+Common styles
+
+*/
+
+/* Split this out as it's a common pattern.
+   At some point, it might make sense to abstract it or add it
+	 at a global .css level. */
+.listResetStyles {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	width: 100%;
+}
+
+/* Description styles markdown content. Note we do not use MDX in this
+   case, so specific styles we want for markdown content will need
+	 to be added here. */
+.description {
```

Comment:
Could we share some of the mdx base content styles?

Reply:
I think it might be technically possible, but I'm a bit hesitant to, cause I feel like it might be a bit of an early abstraction... One notable difference with these styles is the base font-size varies depending on the nesting level of the property in question.

My intent for now was to keep the styles here isolated, especially since these descriptions seem to only contain basic text styling and links (as opposed to more fully-featured markdown use such as block elements). I think if we find ourselves reaching farther to style many other elements (say lists, code blocks, etc) then switching this to use `next-mdx-remote` would probably make sense 👍